### DISCUSSION TITLE
Does basal mass balance include subglacial conduit volume?

### DISCUSSION THREAD

**Discussion by mankoff** [pism/pism#531] [Q&A]
What terms go into this basal mass balance? I assume geothermal heat flux and frictional heating from sliding? What about viscous heat dissipation of surface-runoff routed to the bed?. This is about the same order of magnitude (in Greenland) as frictional heating mass loss or geothermal heat flux (see http://doi.org/10.1038/s41467-021-23739-z).

Is surface runoff routed to the bed? I assume not, because that implies treatment of moulins and a level of detail I did not think existed in PISM.

**Comment by bueler** [comment]
The basal mass balance comes from the basal conservation of energy problem described in https://doi.org/10.3189/2012JoG11J088.  See Fig. 5 and surrounding text.  It is a bit of a mess because of various inequalities constraining the quantities.  For example, temperature of an ice/water mixture does not exceed melting temperature even as you change its enthalpy.  But also refreeze stops once all liquid in the transportable and till subglacial water system is used up.  Etc.

To answer your question, yes geothermal flux is an input.  Frictional heating is computed and contributes to the energy balance computation, thus the basal mass balance.

[_Edit: Not a trustworthy paragraph.  See comments below._]  No, I don't think there is any surface runoff to the bed.  (There was not such back when I was expert on PISM.  I am not expert anymore, however.  I may have lost track.)  I think you are right that there is currently no attempt to keep track of moulins.

Note that, because PISM uses enthalpy, some of the energy other models ascribe to "basal friction" is modeled as viscous dissipation within temperate ice.  Some of the resulting liquid water within the temperate ice is "instantly" transported to the bed according to a "drainage model" (invented by Greve for his polythermal ice work).  Specifically for PISM, from the cited enthalpy paper above, there is drainage to the bed of a portion of the liquid water in the temperate ice.  We have a highly simplified drainage model.  First there is an assumed amount of liquid water within temperate ice below which there is no drainage (`energy.drainage_target_water_fraction`)  Its default value of 0.01 means we allow up to 1% of the temperate ice mixture to be liquid before any drainage occurs.  Above that amount of liquid water there is drainage at a rate controlled by the function shown in Figure 4 of the above reference, and by the parameter `energy.drainage_maximum_rate`.  I suppose my point in describing this is to identify an important contributor to the basal mass balance (i.e. production going into the till and transportable subglacial water fields in PISM), one which you may not already be thinking of.

**Reply by bueler** [reply]
Two more things.

1. Your title has "subglacial conduit volume".  I am not sure what _you_ mean by this.  Within PISM there is no quantity that precisely corresponds to this phrase.  See https://www.pism.io/docs/manual/modeling-choices/subglacier/hydrology.html.  The closest to "conduit volume" is the transportable water thickness `bwat` in the `-hydrology routing` mass conserving model.

2.  I see that I am out of date regarding surface runoff being routed to the bed.  From the above manual page, I see that by setting `hydrology­.surface_input_from_runoff` you can make the top-of-the-ice surface runoff get added to the basal hydrology water amount.  This mechanism seems to exist both for surface runoff provided in a file, and as computed by a temperature index scheme at the surface.  I have limited understanding of `-hydrology steady`.  I should read the docs before answering questions based on what I remember!

**Reply by mankoff** [reply]
The `subglacial conduit volume` is in reality, perhaps not the model. The viscous heat dissipation from the surface runoff melts open conduits. That is, the potential energy `mgz` from the surface is released at the bed. We assume this is used to melt open conduits. This energy is roughly equal (area integrated, in Greenland) to geothermal heat flux which is also roughly equal to frictional heating. Each contributes about 1/3 to the basal mass balance. It sounds like this term is treated properly in the current version of PISM.

**Reply by bueler** [reply]
> The subglacial conduit volume is in reality, perhaps not the model.

Understood!  It is a good exercise for me to try to map your intent/questions to PISM.

> The viscous heat dissipation from the surface runoff melts open conduits.
> That is, the potential energy mgz from the surface is released at the bed.
> We assume this is used to melt open conduits. This energy is roughly equal
> (area integrated, in Greenland) to geothermal heat flux which is also roughly
> equal to frictional heating. Each contributes about 1/3 to the basal mass balance.
> It sounds like this term is treated properly in the current version of PISM.

The energy of surface runoff to the bed is (I believe) _not_ treated properly in PISM.

FYI:  On the other hand, the contribution which temperature-based models compute for "frictional heating" (e.g. as described in the paper by Karlsson et al) may be missing quite a bit of energy.  I put it in quotes because what one is really trying to capture is all of the dissipated energy of the precipitation at the top of the glacier which heads back down to sea level (e.g. at an outlet glacier) via any route, whether ice flow itself or a moulin, or liquid flow on the surface.  Some dissipation is literally in sliding at the base, but some occurs as large strain rates in near-basal ice.  Just like moulin runoff, all parts of a glacier have their energy from gravitational potential.  Once we switched PISM to an enthalpy model we saw that there was quite a bit of the gravitational potential being shoved into 0C (i.e. temperate) ice.  That is, potential energy is converted to liquid water fraction within the ice, at least comparable to basal melt on the lower surface of the ice.

**Comment by aaschwanden** [comment]
> On Dec 20, 2023, at 09:35, Ed Bueler ***@***.***> wrote:
> 
> 
> Two more things.
> 
> Your title has "subglacial conduit volume". I am not sure what you mean by this. Within PISM there is no quantity that precisely corresponds to this phrase. See https://www.pism.io/docs/manual/modeling-choices/subglacier/hydrology.html. The closest to "conduit volume" is the transportable water thickness bwat in the -hydrology routing mass conserving model.
> 
> I see that I am out of date regarding surface runoff being routed to the bed. From the above manual page, I see that by setting hydrology­.surface_input_from_runoff you can make the top-of-the-ice surface runoff get added to the basal hydrology water amount. This mechanism seems to exist both for surface runoff provided in a file, and as computed by a temperature index scheme at the surface. I have limited understanding of -hydrology steady. I should read the docs before answering questions based on what I remember!
> 

That’s correct. I have been using this mechanism to allow surface meltwater runoff to be routed to the base and then transported to marine termini where it is used to calculate frontal melt.
